charts/prometheus-mysql-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for prometheus mysql exporter with cloudsqlproxy
 name: prometheus-mysql-exporter
-version: 1.4.0
+version: 1.4.1
```

new feature should raise minor version.
```suggestion
version: 1.5.0
```